modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/leases/LeaseTracker.java

```diff
@@ -214,7 +214,6 @@ public CompletableFuture<LeaseMeta> getPrimaryReplica(ReplicationGroupId replica
                 return completedFuture(lease);
             }
 
-            // TODO: https://issues.apache.org/jira/browse/IGNITE-19532 Race between meta storage safe time publication and listeners.
```

Comment:
Did you fix it? Or it was here by mistake?

Reply:
This is a leftover from other ticket